cfgov/core/utils.py

```diff
@@ -70,8 +67,17 @@
 
 def should_interstitial(url: str) -> bool:
     match = LINK_PATTERN.match(url)
+
+    # If this is another link to .gov and it's not a CFPB link, do not
+    # interstitial.
     if match.group("domain").endswith(".gov") and NON_CFPB_LINKS.match(url):
```

Comment:
This may not be germane to this PR, but can you remove the `and NON_CFPB_LINKS.match(url)`  clause here? `should_interstitial is already called from within an `elif` that checks the truthiness of `NON_CFPB_LINKS.match(href)`.

Reply:
Yup, as good a time as any while I'm here.